settings.gradle

```diff
@@ -33,9 +33,9 @@ buildscript {
 }
 
 plugins {
-    id "com.gradle.enterprise" version "3.8.1"
-    id "com.gradle.enterprise.test-distribution" version "2.2.2"
-    id "com.gradle.common-custom-user-data-gradle-plugin" version "1.6.2" apply false
+    id "com.gradle.enterprise" version "3.9"
+    id "com.gradle.enterprise.test-distribution" version "2.2.3"
+    id "com.gradle.common-custom-user-data-gradle-plugin" version "1.6.5" apply false
```

Comment:
why apply false here? there's not reason to conditionally apply this on line 44
i know this isn't something related to your PR, but might be worth removing in a separate PR.

Reply:
could be that they have some conditional logic handling that
https://github.com/jprinet/kotlin/blob/3ee8aabe4cbae7ddc5175e85a440a9ae42677465/settings.gradle#L44

Reply:
i mean that there's no reason to have that. it removes information from the public scans which are also useful.

Reply:
this is a wider scope than the current PR imo.
We might discuss that with Jetbrains on our Slack channel

Reply:
👍